Challengue_22/Fanny_Gabriela_Becerra_Hernandez/main.js

```diff
@@ -0,0 +1,128 @@
+/***************************************************************************************************
+ * Name: ARRAYS_LoopsInJavascript
+ * Author:
+ * Description: This script tests for loops for an array.
+ * *************************************************************************************************
+ * LastModifiedBy     |     LastModifiedDate     |     Modification Id     |     Change description
+ * - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
+ * Fanny Becerra      |     26-Jun-2020          |     v0.1                |     Beta version
+ */
+
+let counterFor = 0,
+    counterForIn = 0,
+    counterForOf = 0,
+    counterWhile = 0,
+    fruitsArray = ["Aguacatito", "Berenjenita", "Pepinito feliz", "Tomatito bebé"],
+    index = 0,
+    table = null,
+    td = null,
+    tr = null,
+    txt = '';
+
+/** @author:      Fanny Becerra.
+ *   @param:       None.
+ *   @return:      None.
+ *   @description: this function tests while loop.
+ */
+function funcWhile() {
+    table = document.getElementById('while');
+    console.log('WHILE');
+    if (counterWhile == 0) {
+        while (index < fruitsArray.length) {
+            console.log(fruitsArray[index]);
+            printRow(table, fruitsArray[index]);
+            index++;
+        }
+        index = 0;
+        counterWhile++;
+
+    } else {
+        deleteRows(table);
+        counterWhile = 0;
+    }
+}
+
+/** @author:      Fanny Becerra.
+ *   @param:       None.
+ *   @return:      None.
+ *   @description: this function tests for loop.
+ */
+function funcFor() {
+    table = document.getElementById('for');
+    console.log('FOR');
+    if (counterFor == 0) {
+        for (index; index < fruitsArray.length; index++) {
```

Comment:
Intenta con lo siguiente:
for(let index = 0, fruitsLenght = fruitsArray.length; index < fruitsLenght  ; index += 1 )